README.md

```diff
@@ -17,14 +47,14 @@ CUDA installation guides: [Linux](https://docs.nvidia.com/cuda/cuda-installation
 To run the CPU version in parallel, [OpenMP](https://www.openmp.org/resources/openmp-compilers-tools/) must be installed. 
 OpenMP comes with recent gcc versions. However, OpenMP is not necessary if one does not want to execute programs in parallel on the CPU.
 
-### Install rootppl
+#### Install rootppl
 To install rootppl for the current user, first clone this repository and change directory to the rootppl folder. Then run:
```

Comment:
Change to `rootppl` in typewriter font.

Reply:
Sure, I can change it. Note, however, that I have not made any changes to the RootPPL section (written by Joey).